docs/docusaurus/docs/cloud/gx_cloud_architecture_and_deployment_patterns.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_label: 'GX Cloud architecture and deployment patterns'
+title: 'GX Cloud architecture and deployment patterns'
+description: Learn more about the components of GX Cloud architecture and common GX deployment patterns
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+GX Cloud provides a secure, fully-managed SaaS solution to monitor, manage, and share your data quality ecosystem. GX Cloud offers flexible deployment patterns that accommodate your environment and supported data sources.
+
+## GX Cloud architecture
+
+GX Cloud's architecture is depicted in the below diagram and is comprised of the following key components:
+
+![GX Cloud Architecture and Components](./architecture_deployment_images/gx_cloud_architecture_components.png)
+
+- **Cloud storage for data quality configuration, metadata, and history.** GX Cloud provides cloud storage for the Data Source, Data Asset, Expectation Suite, and Checkpoint configurations that comprise your GX workflows. GX Cloud also stores your Validation run histories and Data Asset descriptive metrics.
+
+- **GX Cloud Web App.** GX Cloud includes a web app that is browser- and platform-independent. The GX Cloud Web App enables you to manage and validate your data without running Python code, and provides shared visibility into your Validation Results and Checkpoint run history.
+
+- **GX Cloud REST API.** The GX Cloud REST API provides an interface to programmatically access and manage GX Cloud data and configurations. Both the GX open source software (OSS) client and the GX Agent utilize the GX Cloud REST API to query data from and send data to GX Cloud. The GX Cloud REST API is not currently publicly documented.
```

Comment:
"Is not currently publicly documented," implies this, but should we add explicit language that the REST API is not intended for use outside of the Python client?

Reply:
Incorporated!